main.go

```diff
@@ -76,7 +76,8 @@ func main() {
 
 	//Close the sensors
 	for _, sensor := range sensors {
-		sensor.Close()
+		s := sensor.(io.Closer)
+		s.Close()
```

Comment:
I'm not familiar what casting sensor to an (io.Closer) does...reading through here: https://golang.org/pkg/io/#Closer

Does it make sure I/O work is cleaned up within the sensor structure? In any case, can we somehow move this cast into `phidget.Phidgets` so it's not needed by library consumers?

Reply:
This was a hang-over from when I was using type Phidget struct {}, rather than an interface. It's no longer necessary, so I've reinstated the original behaviour.